website/src/styles/mdcontainer.scss

```diff
@@ -123,3 +123,12 @@
 .mdContainer summary{
     @apply cursor-pointer;
 }
+
+.mdContainer  .faqWrapper h3{
+    @apply text-lg font-semibold text-main my-3;
+    border-top: 1px solid #ddd;
+    margin-top: 1.75rem;
+    padding-top: 1.25rem;
+    margin-bottom:0;
```

Comment:
Would it be better to not mix the use of tailwind classes and raw CSS and stick with one type as much as possible?